### PR TITLE
1637083: add unset sections

### PR DESCRIPTION
**Pull request by tlhmerry9**

### CODE REVIEW

**Comment by tlhmerry9** [comment]
To Test (this branch was created from the subscription-manager-1.28 branch)

man syspurpose/man/syspurpose.8

File: syspurpose/man/syspurpose.8 [hunk 1]
```diff
@@ -48,6 +48,9 @@ Add values to the add-ons property
 \fBsyspurpose remove-addons \fIVALUE\fP...\fP
 Remove values from the add-ons property
 .TP
+\fBsyspurpose unset-addons \fIVALUE\fP...\fP
+Clear set add-ons property
```
Comment (jirihnidek) [inline syspurpose/man/syspurpose.8:52]: I would write "Clear the system addons property". addons property is a set, but it looks confusing.

**Review by jirihnidek — CHANGES_REQUESTED** [review]
Hi, thanks for PR. I have one comment in the this man page. BTW: We use two very similar words "add-ons" and "addons" for the same thing. Shouldn't we use only "addons"?

**Comment by jirihnidek** [comment]
Oh, I can see that this PR has conflict too. Please resolve this conflict.

**Comment by tlhmerry9** [comment]
Accidentally merged this when resolving the conflict

**Comment by jirihnidek** [comment]
> Accidentally merged this when resolving the conflict

Please, be more careful next time. Merged commit looks OK.

**Comment by tlhmerry9** [comment]
I will most definitely be more careful!

________________________________

**Pull request merged**